app/views/ev_map/show.html.erb

```diff
@@ -1,16 +1,18 @@
 <div id="mapid"></div>
 
 <script>
-  var mymap = L.map('mapid').setView([<%= @ev_search_lat %>, <%= @ev_search_lon %>], 13);
+  const mymap = L.map('mapid').setView([<%= @ev_search_lat %>, <%= @ev_search_lon %>], 13);
 
   L.tileLayer('https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={accessToken}', {
     attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
     maxZoom: 20,
     id: 'mapbox.streets',
     accessToken: 'pk.eyJ1IjoibWFyeWFncmFjZSIsImEiOiJjanM0eG9qMmIwOXlhNDVuaTNuOXMzcXd1In0.Y3D7BRb8btkvg5JV_6PZaQ'
```

Comment:
Need to pull this token out of the view for sure before we get popped for it. We have a way to bring it in per Mary